todo-be/modules/hello.py

```diff
@@ -0,0 +1,6 @@
+from app import app
+from flask import jsonify
+
+@app.route('/')
+def hello():
```

Comment:
So are we doign only hello world? Or we are goimg to update it with the todo thing later?

Reply:
I'll update it with todo later